src/pf-3-form-fields/select-styles.js

```diff
@@ -0,0 +1,78 @@
+const customStyles = {
+  placeholder: provided => ({
+    ...provided,
+    lineHeight: 25,
+    padding: '0 6px',
+    color: 'rgb(77, 82, 88)',
+    fontWeight: '600',
+    paddingLeft: 0,
+    marginLeft: 0,
+  }),
+  option: (provided, { isFocused, isSelected }) => ({
+    ...provided,
+    'background-color': isSelected ? '#0088ce' : isFocused ? '#def3ff' : '#ffffff',
+    'border-color': isSelected ? '#0088ce' : isFocused ? '#bee1f4' : 'transparent',
+    'border-style': 'solid',
+    'border-width': '1px 0',
+    color: isSelected ? '#fff' : isFocused ? '#4d5258' : provided.color,
+    padding: '1px 10px',
+    fontWeight: '400',
+    cursor: 'pointer',
+  }),
+  dropdownIndicator: provided => ({
+    ...provided,
+    color: 'rgb(77, 82, 88)',
+  }),
+  container: provided => ({
+    ...provided,
+    transition: 'border-color ease-in-out .15s, box-shadow ease-in-out .15s',
+    fontFamily: '"Open Sans", Helvetica, Arial, sans-serif',
+    padding: 0,
+    border: '1px solid transparent',
+    'border-color': 'rgb(187, 187, 187)',
+  }),
+  singleValue: provided => ({
+    ...provided,
+    height: 25,
```

Comment:
just curious, there is no need for 'px'?

Reply:
nope it will be added automatically. See: https://reactjs.org/docs/dom-elements.html#style